x-pack/plugins/enterprise_search/public/applications/shared/kibana_chrome/generate_breadcrumbs.ts

```diff
@@ -118,3 +119,9 @@ export const useWorkplaceSearchBreadcrumbs = (breadcrumbs: Breadcrumbs = []) =>
     { text: WORKPLACE_SEARCH_PLUGIN.NAME, path: '/' },
     ...breadcrumbs,
   ]);
+
+export const useEnterpriseSearchContentBreadcrumbs = (breadcrumbs: Breadcrumbs = []) =>
+  useEnterpriseSearchBreadcrumbs([
+    { text: ENTERPRISE_SEARCH_CONTENT_PLUGIN.NAV_TITLE, path: '/' },
```

Comment:
I thought about reusing:
```
  i18n.translate('xpack.enterpriseSearch.content.searchIndices.content.breadcrumb', {
    defaultMessage: 'Content',
  })
```
here instead of the `ENTERPRISE_SEARCH_CONTENT_PLUGIN.NAV_TITLE`, but wasn't sure which was preferable.

Reply:
Using `ENTERPRISE_SEARCH_CONTENT_PLUGIN.NAV_TITLE` here is fine. 

This may re-shape again most likely once we add new pages with overlapping breadcrumbs. Since I am not sure what will be the shape, it is fine as it is.